cmd/destroy/destroy.go

```diff
@@ -240,6 +240,10 @@ func (dc *destroyCommand) getDestroyer(opts *Options) (destroyInterface, error)
 			}
 		}
 
+		if err := manifest.ExpandEnvVars(); err != nil {
```

Comment:
I have some concerns about expanding envvars here. The `ExpandEnvVars` method is not fully idempotent, we've had issues from some users regarding the escaping of the envvars and this would add to that pile. 

Are we sure that this is getting called only once?

Reply:
yes, you are right. We might have problems if we call twice that func. I didn't take into account that we could expand twice the manifest if we deploy directly locally. I refactored it to avoid it in 1e75def29eaa4099a418c61821084d9ff50f2a13. Thanks!!